modules/nodes-nodes-rebooting-infrastructure.adoc

```diff
@@ -3,26 +3,15 @@
 // * nodes/nodes-nodes-rebooting.adoc
 
 [id="nodes-nodes-rebooting-infrastructure_{context}"]
-= Understanding infrastructure node rebooting
+= About rebooting nodes running critical infrastructure
 
-Infrastructure nodes are nodes that are labeled to run pieces of the
-{product-title} environment. Currently, the easiest way to manage node reboots
-is to ensure that there are at least three nodes available to run
-infrastructure. The nodes to run the infrastructure are called *master* nodes.
+When rebooting nodes that host critical {product-title} infrastructure components, such as router pods, registry pods, and monitoring pods, ensure that there are at least three nodes available to run these components.
 
-The scenario below demonstrates a common mistake that can lead
-to service interruptions for the applications running on {product-title} when
-only two nodes are available.
+The following scenario demonstrates how service interruptions can occur with applications running on {product-title} when only two nodes are available:
 
 - Node A is marked unschedulable and all pods are evacuated.
-- The registry pod running on that node is now redeployed on node B. This means
-node B is now running both registry pods.
+- The registry pod running on that node is now redeployed on node B. Node B is now running both registry pods.
 - Node B is now marked unschedulable and is evacuated.
-- The service exposing the two pod endpoints on node B, for a brief period of
-   time, loses all endpoints until they are redeployed to node A.
+- The service exposing the two pod endpoints on node B, for a brief period of time, loses all endpoints until they are redeployed to node A.
```

Comment:
Maybe bring `for a brief period of time` to the front or end of this sentence to avoid separating `The service [...] loses` further than necessary.